elide-core/src/test/java/com/yahoo/elide/core/PersistentResourceTest.java

```diff
@@ -2167,6 +2176,57 @@ public void testRelationChangeSpecType() {
             assertTrue(model.updateRelation("child", null));
     }
 
+    @Test
+    public void testPatchRequestScope() {
+        DataStoreTransaction tx = mock(DataStoreTransaction.class);
+        PatchRequestScope parentScope =
```

Comment:
Does the code actually nest patch scopes like this?  I thought patch was the parent scope - but child scopes were always normal request scopes.

Reply:
Yes, they are nested.  That is what this tests.  See

https://github.com/yahoo/elide/blob/master/elide-core/src/main/java/com/yahoo/elide/extensions/JsonApiPatch.java#L65